### PR TITLE
chore(pack-up): remove from monorepo (v5)

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

* removes pack-up from the monorepo

### Why is it needed?

* it's got it's own space to breath and live

### How to test it?

* see https://github.com/strapi/pack-up

### Related issue(s)/PR(s)

* resolves DX-1342

### CODE REVIEW

**Review by Convly — APPROVED** [review]
LGTM, see you around pack-up

image

**Review by jhoward1994 — APPROVED** [review]

**Review by innerdvations — APPROVED** [review]
LGTM, one less package to build 5x a day 😆

(warning: I did not QA this so make sure someone else makes sure it doesn't suddenly break the build or anything)

**Comment by joshuaellis** [comment]
@innerdvations ive had to do some changes for type fixes, therefore it should really be reviewed again, soz

File: packages/core/admin/server/src/services/role.ts [hunk 1]
```diff
@@ -201,7 +201,7 @@ const deleteByIds = async (ids = [] as Data.ID[]): Promise<AdminRole[]> => {
 
   await getService('permission').deleteByRolesIds(ids);
 
-  const deletedRoles = [];
+  const deletedRoles: any[] = [];
   for (const id of ids) {
```
Comment (innerdvations) [inline packages/core/admin/server/src/services/role.ts:205]:
```suggestion
  const deletedRoles: AdminRole[] = [];
```

File: packages/core/content-manager/server/src/services/utils/configuration/layouts.ts [hunk 2]
```diff
@@ -67,9 +67,9 @@ function syncLayouts(configuration: any, schema: any) {
   // backward compatibility with when relations were on the side of the layout
   // it migrates the displayed relations to the main edit layout
   const elementsToReAppend = [...cleanEditRelations];
-  let cleanEdit = [];
+  let cleanEdit: any[] = [];
   for (const row of edit) {
-    const newRow = [];
+    const newRow: any[] = [];
```
Comment (innerdvations) [inline packages/core/content-manager/server/src/services/utils/configuration/layouts.ts:73]:
```suggestion
  let cleanEdit: unknown[] = [];
  for (const row of edit) {
    const newRow: unknown[] = [];
```

**Review by innerdvations — COMMENTED** [review]
Just some tiny changes because I'm allergic to adding any more `any` to the codebase. The `as` gets a pass only because it would require changing the actual logic and that's for another PR another day.

File: packages/core/admin/server/src/services/permission/queries.ts [hunk 3]
```diff
@@ -28,7 +28,7 @@ export const deleteByRolesIds = async (rolesIds: Data.ID[]): Promise<void> => {
  * @param ids ids of permissions
  */
 export const deleteByIds = async (ids: Data.ID[]): Promise<void> => {
-  const result = [];
+  const result: any[] = [];
```
Comment (innerdvations) [inline packages/core/admin/server/src/services/permission/queries.ts:31]:
```suggestion
  const result: unknown[] = [];
```

File: packages/core/admin/server/src/services/role.ts [hunk 4]
```diff
@@ -201,7 +201,7 @@ const deleteByIds = async (ids = [] as Data.ID[]): Promise<AdminRole[]> => {
 
   await getService('permission').deleteByRolesIds(ids);
 
-  const deletedRoles = [];
+  const deletedRoles: any[] = [];
```
Comment (innerdvations) [inline packages/core/admin/server/src/services/role.ts:204]:
```suggestion
  const deletedRoles: AdminRole[] = [];
```

File: packages/core/content-manager/server/src/services/utils/configuration/layouts.ts [hunk 5]
```diff
@@ -67,9 +67,9 @@ function syncLayouts(configuration: any, schema: any) {
   // backward compatibility with when relations were on the side of the layout
   // it migrates the displayed relations to the main edit layout
   const elementsToReAppend = [...cleanEditRelations];
-  let cleanEdit = [];
+  let cleanEdit: any[] = [];
```
Comment (innerdvations) [inline packages/core/content-manager/server/src/services/utils/configuration/layouts.ts:70]:
```suggestion
  let cleanEdit: unknown[] = [];
```

File: packages/core/content-manager/server/src/services/utils/configuration/layouts.ts [hunk 6]
```diff
@@ -67,9 +67,9 @@ function syncLayouts(configuration: any, schema: any) {
...
   for (const row of edit) {
-    const newRow = [];
+    const newRow: any[] = [];
```
Comment (innerdvations) [inline packages/core/content-manager/server/src/services/utils/configuration/layouts.ts:72]:
```suggestion
    const newRow: unknown[] = [];
```

**Review by innerdvations — APPROVED** [review]

**Pull request merged**